sdk/servicebus/azure-servicebus/tests/stress_tests/stress_test_base.py

```diff
@@ -81,14 +84,21 @@ def __init__(self,
 
 
     # Plugin functions the caller can override to further tailor the test.
-    def OnSend(self, state, sent_message):
-        '''Called on every successful send'''
+    def OnSend(self, state, sent_message, sender):
+        '''Called on every successful send, per message'''
         pass
 
-    def OnReceive(self, state, received_message):
-        '''Called on every successful receive'''
+    def OnReceive(self, state, received_message, receiver):
+        '''Called on every successful receive, per message'''
         pass
 
+    def OnReceiveBatch(self, state, batch, receiver):
```

Comment:
just curious - why dont we have snake case here?

Reply:
Because I'm not nearly as consistent as I should be, and this utility initially started as a side-script I was using for personal testing.  Good callout, will normalize++